specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -229,9 +237,20 @@ The availability of the extension will be managed by JupyterLab since it is goin
 In terms of security, Jupyter uses tornado to ensure only authorized user can request the Jupyter server
 You can read more [here](https://jupyter-notebook.readthedocs.io/en/stable/security.html#).
 
-VDK Control Service uses authentication in REST API, based on OAuth2 To authenticate specify OAuth2 access token as Authorization/Bearer Header.
-The testing installation uses (Staging) CSP Authentication provider. To get access token you need refresh or access token To get refresh token go to https://console-stg.cloud.vmware.com/csp/gateway/portal/#/user/tokens
-
+VDK Control Service uses authentication in REST API, based on OAuth2 To authenticate specify OAuth2 access token as an Authorization/Bearer Header.
+Access token would be generated using one of 2 approaches depending on how Jupyter server is deployed:
+
+* Standalone JupyterLab (catered for individual users in their local environments):
+   * Login:
+       Initiates the [OAuth2 Authorization Flow](https://tools.ietf.org/html/rfc6749#section-4.1) upon selecting "Login" from the VDK dropdown with callback to the server
+       The server would finish the authorization flow leveraging [tornado Oauth2Mixin](https://www.tornadoweb.org/en/stable/auth.html)
+       Access token, once received, is securely stored within JupyterLab backend.
+   * Logout: Access token data is deleted from the backend
+* JupyterHub Deployment (or any other similar multi-user, centralized deployments, with users already authenticated):
+   * Login: Leverages the access token generated during JupyterHub's authentication.
```

Comment:
I don't understand this. 

Can you add more details here. 


Do they need to login before they can start using jupyter at all?

Reply:
I will try to provide more details . But the we are running Jupiter as part of managed installation that can start different notebooks in separate containers usually and since we are in the same session we can access the access token. I will provide more details and link

Reply:
cool

Reply:
Yeah, here I was thinking the same that the information is too little. But i do not thinking overpopulating the VEP would be good too. Maybe adding links to the JupyterHub documentation is best

Reply:
I provided more details